tests/nnapi/specs/Ex/cos_ex_dynamic_nnfw.mod.py

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2018 The Android Open Source Project
+# Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# refer to tanh_v1_dynamic.mod.py about the structore
+
+# This adds reshape as the first op in a model and
+# returns output of reshape, which is dynamic tensor
+
+'''
+Testing Cos op when the input is dynamic.
+      input [1, 2, 3]  shape [3]  (value of shape will be [1, 2, 3])
+          |             |
+          +-------------+
+          |
+       Reshape (added by DynamicInputGenerator since it generates its output to be dynamic)
+          |
+          | dynamic tensor at compilation time but the shape will be [1, 2, 3] at execution time
+          |
+         Abs
```

Comment:
maybe `Cos`?

Reply:
@hyunsik-yoon I modified. Is there anything to add?